src/Context/RawContext.php

```diff
@@ -114,7 +114,7 @@ public function iSwitchToIFrameWithCSSLocator($locator)
    * @return bool
    * @throws Exception
    */
-    public function spin($lambda, $wait = 240)
+    public function spin($lambda, $wait = 500)
```

Comment:
Given this repo (`behat-tools`) is used by several repos (not just `frost-qa`), I think it'd be better to pass in the increased `$wait` as an argument to `spin()` rather than changing the default value in `spin()` itself.

The `spin()` method is declared as `spin($lambda, $wait)` so it should be easy to pass in a non-default `$wait` value for specific places we need it.